app/models/group.rb

```diff
@@ -0,0 +1,23 @@
+class Group < ApplicationRecord
+  belongs_to :user
+  has_many :transacs
+
+  validates :name, presence: true, uniqueness: true
+  validates :icon, presence: true
+
```

Comment:
- [ ] Please as in the  diagram the group and transacs should have many to many relationship currently they have on to many relationship.
![image](https://user-images.githubusercontent.com/74744787/227728941-408863f0-9b16-4656-a5ff-a8d995bcd464.png)

Reply:
Thank you for the good review, I will implement it now